src/gluonts/model/deepar/_network.py

```diff
@@ -1140,6 +1209,8 @@ def hybrid_forward(
         Tensor
             Predicted samples
         """
+        if past_past_feat_dynamic_real is not None:
+            past_past_feat_dynamic_real.mean()
```

Comment:
Why is this required?

Reply:
Thanks for spotting this. Will remove it, this was to test whether the data gets actually passed.